tests/RegressionTests_gaea.intel.log

```diff
@@ -1,3 +1,30 @@
+<<<<<<< HEAD
```

Comment:
This file has merge conflicts, but will be overwritten by auto-rt - therefore no problem.

Reply:
Yes, I did that on purpose as I knew it would be overwritten.